pavelib/prereqs.py

```diff
@@ -38,23 +39,21 @@
     PYTHON_REQ_FILES.append(PRIVATE_REQS)
 
 
+def str2bool(s):
+    s = str(s)
+    return s.lower() in ('yes', 'true', 't', '1')
+
+
 def no_prereq_install():
     """
     Determine if NO_PREREQ_INSTALL should be truthy or falsy.
     """
-    vals = {
-        '0': False,
-        '1': True,
-        'true': True,
-        'false': False,
-    }
+    return str2bool(os.environ.get('NO_PREREQ_INSTALL', False))
```

Comment:
os.environ.get returns a string, and str2bool kind of expects one, so the default value should be a string.  Same for the similar call on line 56.